lib/matplotlib/axes/_base.py

```diff
@@ -117,6 +117,9 @@ def __call__(self, ax, renderer):
             self._transform - ax.figure.transSubfigure)
 
 
+_FORMAT_UNSET = 'None'
```

Comment:
should that not have been an `object()` placeholder, to avoid confusion with other meanings of the string "none"?

Reply:
@anntzer When linestyle or marker are not specified in a format string, they are deactivated. "None" actually serves as value down the road in `set_linestyle()`. We cannot have a proper sentinel (or would have to convert it again down the road).

My intention was to make it more clear what the semantics of the value are. Otherwise
https://github.com/matplotlib/matplotlib/blob/2219872528f2f7d1557126b15451489918225318/lib/matplotlib/axes/_base.py#L474

would be `and val != 'None'`. Which is less understandable, and freaks me out because of the case-insensitivity of our 'none' string handling. I was about to write `and val.lower() != 'none'` since I didn't follow closely where that was coming from, but then again, *val* can be a 4-tuple for colors, which you cannot `lower()`.

But thinking again, it's better to be explicit with 'none' and not pretend to have a sentinel, that actually has hidden meaning.

Reply:
thanks for taking care of that.